components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/attestation_bootloader_data.h

```diff
@@ -27,19 +27,29 @@
 extern "C" {
 #endif
 
-/* Temp Shared data area between bootloader and runtime firmware */
+#define IMPL_ID_DATA_SIZE  S_IMPL_ID_DATA_SIZE
```

Comment:
what is the reason for having two versions of these macros? the one with `S_` and the one without?